### ISSUE TITLE
Aligning workplans & FAST

### ISSUE THREAD

**Issue by ramona-godbole** [USAID-OHA-SI/TeamTracking#64]
SCC collaborating with SIEI to develop workplan template that aligns with new financial monitoring categories/FAST

**Comment by ramona-godbole** [comment]
Update 1/31:  Draft workplan template developed and submitted to Clint/FO for initial feedback before moving forward. 

[https://docs.google.com/document/d/1UqVCAijGY-uFEppsKaiepbvD2oBZLYY6cmaFuoyFFMo/edit](url)
[https://docs.google.com/spreadsheets/d/1bnqAND5bogpOxjRt-Qho0FZR6KO3C05U-Ow6Qm0thJk/edit#gid=1397394923](url)

Next Steps to operationalize: 
-check in with FO, SGAC, other stakeholders
-Work with partner to get feedback on feasibility/openness to this approach
-Liaise with Table 6, SRE, targets, etc. stakeholders involved in minimum required
-Link with technical folks working on guidance/language document to marry two processes 
-Reach out to a few A/CORs at HQ in short term for feedback (Colette)
-Review S. Africa & Zambia workplans as best practices for potential areas to incorporate (PTCs could support)
-Discuss with Elan and (potentially) SGAC financial operations stakeholders expectations around sub-recipient level budgeting 
-Review near-final product with OAA?

**Comment by ramona-godbole** [comment]
Update Feb 28:  Aaron working on refining existing DataPack/FAST datasets to be data sources to produce individualized mechanism-specific templates

**Comment by achafetz** [comment]
@lucasrachel, @jaliasd did you have a conversation Wednesday about who could compile an updated Data Pack folder on Google Drive (and also convert the xlsb files xlsx)?

**Comment by lucasrachel** [comment]
No, we did not come to decisions on Wednesday. Would this be a reasonable distribution:
Group 1 countries: Josh takes lead
Group 2 countires: Aaron or Gina leads, as Jacob is tied up with a lot leading up to TDY
Group 3 countries: Gina takes lead

Josh, since you may have already done this for your countries, could you set up file structure in Google Drive? Thanks

**Comment by achafetz** [comment]
Sounds like a plan. Thanks, @lucasrachel.

@jaliasd if you could point us to a folder on Google Drive, I can add the Group 2 OUs and @gsarfaty can add Group 3 (please :smiley: ) Work for everyone? I'll pull them down and consolidate using `tameDP`

**Comment by jaliasd** [comment]
DPs go [here](https://drive.google.com/open?id=1oT_hLImUPkjHCbEogxBNssbaC3xGoE-0)

- [ ] Angola
- [ ] Botswana
- [ ] Eswatini
- [ ] Lesotho
- [ ] Malawi
- [ ] Mozambique
- [ ] Namibia
- [ ] South Africa
- [ ] Zambia
- [ ] Zimbabwe
- [ ] Caribbean Region
- [ ] Central America Region
- [ ] Dominican Republic
- [ ] Guyana
- [ ] Haiti
- [ ] Western Hemisphere Region (WHR)​
- [ ] Burundi
- [ ] Cameroon
- [ ] Côte   d'Ivoire
- [ ] Democratic   Republic of the Congo
- [ ] Ethiopia
- [ ] Ghana
- [ ] Kenya
- [ ] Nigeria
- [ ] Rwanda
- [ ] South   Sudan
- [ ] Tanzania
- [ ] Uganda
- [ ] West Africa Region
- [ ] Asia   Region
- [ ] Asia Regional   Program (ARP)
- [ ] Burma
- [ ] Cambodia
- [ ] Central   Asia Region (CAR)
- [ ] India
- [ ] Indonesia
- [ ] Papua   New Guinea
- [ ] Russia
- [ ] Ukraine
- [ ] Vietnam

**Comment by achafetz** [comment]
Data Packs that we had with the expection of the ones we had issues (Nigeria and Ethiopia) with were consolidated and [posted to Google Drive](https://drive.google.com/open?id=1cpagwgP9m6DMIInrlyBdsgW0X-gRTnau).

Next step is to create output a file that consolidates the FY20 targets to [look like the output from the workplan](https://docs.google.com/spreadsheets/d/1bnqAND5bogpOxjRt-Qho0FZR6KO3C05U-Ow6Qm0thJk/edit#gid=1431197463) (ie determine which values to take as the total numerator and aggregate by indicator and mechanism)

**Comment by gsarfaty** [comment]
> Sounds like a plan. Thanks, @lucasrachel.
> 
> @jaliasd if you could point us to a folder on Google Drive, I can add the Group 2 OUs and @gsarfaty can add Group 3 (please  ) Work for everyone? I'll pull them down and consolidate using `tameDP`

Missed notifications for this today but looking it over, seems it was handled by @achafetz. If updates or future assistance is needed, let me know.  My notification settings are updated to ensure time sensitive updates reach my work inbox.

**Comment by achafetz** [comment]
@ramona-godbole, attached below is the workplan output as a result of running `tameDP` and `fastR` to extract the consolidated data from the tools we have available. [Code available on GitHub.](https://github.com/USAID-OHA-SI/ProjectPiecemeal/blob/master/R/piecemeal_cop19.R)

[COP19_WorkPlan_TargetsAndBudget_20190304](https://docs.google.com/spreadsheets/d/1bAhzz3aZRB4siedJeGzYTUut3q-F8Z55ojGcWYhEOfc/edit?usp=sharing)

|                   Operating Unit | Targets | Budget |
|---------------------------------:|:-------:|:------:|
|                           Angola |    X    |    X   |
|            Asia Regional Program |         |        |
|                         Botswana |    X    |    X   |
|                            Burma |         |        |
|                          Burundi |    X    |    X   |
|                         Cambodia |         |        |
|                         Cameroon |    X    |    X   |
|                 Caribbean Region |         |        |
|           Central America Region |         |        |
|              Central Asia Region |         |        |
|                    Cote d'Ivoire |    X    |    X   |
| Democratic Republic of the Congo |    X    |    X   |
|               Dominican Republic |         |        |
|                         Eswatini |    X    |    X   |
|                         Ethiopia |         |    X   |
|                            Ghana |    X    |        |
|                            Haiti |         |        |
|                            India |         |        |
|                        Indonesia |         |        |
|                            Kenya |    X    |    X   |
|                          Lesotho |    X    |    X   |
|                           Malawi |    X    |    X   |
|                       Mozambique |    X    |    X   |
|                          Namibia |    X    |    X   |
|                          Nigeria |         |    X   |
|                 Papua New Guinea |         |        |
|                           Rwanda |    X    |    X   |
|                     South Africa |    X    |    X   |
|                      South Sudan |    X    |    X   |
|                         Tanzania |    X    |    X   |
|                           Uganda |    X    |    X   |
|                          Ukraine |    X    |    X   |
|                          Vietnam |    X    |    X   |
|                           Zambia |    X    |    X   |
|                         Zimbabwe |    X    |    X   |
